catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/BrickValueParameterTest.java

```diff
@@ -196,6 +475,468 @@ public void testSoundBricksDefaultValues() {
+		checkIfBrickShowsText(DroneTurnRightBrick.class, "20.0");
+
+		checkIfBrickShowsText(DroneSwitchCameraBrick.class, R.string.brick_drone_switch_camera);
+	}
+
+	@Test
+	public void testPhiroBricksDefaultValues() {
+		openCategory(R.string.category_phiro);
+
+		checkIfBrickShowsText(PhiroMotorMoveForwardBrick.class, R.string.brick_phiro_motor_forward_action);
+		checkIfBrickShowsText(PhiroMotorMoveForwardBrick.class, R.string.phiro_motor_speed);
+		checkIfBrickShowsText(PhiroMotorMoveForwardBrick.class, R.string.percent_symbol);
+		checkIfBrickShowsText(PhiroMotorMoveForwardBrick.class, "100");
+		checkIfBrickShowsSpinnerWithEditTextOverlayWithText(PhiroMotorMoveForwardBrick.class,
+				R.id.brick_phiro_motor_forward_action_spinner,
+				R.string.phiro_motor_left);
+
+		checkIfBrickShowsText(PhiroMotorMoveBackwardBrick.class, R.string.brick_phiro_motor_backward_action);
+		checkIfBrickShowsText(PhiroMotorMoveBackwardBrick.class, R.string.phiro_motor_speed);
+		checkIfBrickShowsText(PhiroMotorMoveBackwardBrick.class, R.string.percent_symbol);
+		checkIfBrickShowsText(PhiroMotorMoveBackwardBrick.class, "100");
+		checkIfBrickShowsSpinnerWithEditTextOverlayWithText(PhiroMotorMoveBackwardBrick.class,
+				R.id.brick_phiro_motor_backward_action_spinner,
+				R.string.phiro_motor_left);
+
+		checkIfBrickShowsText(PhiroMotorStopBrick.class, R.string.phiro_motor_stop);
+		checkIfBrickShowsSpinnerWithEditTextOverlayWithText(PhiroMotorStopBrick.class,
+				R.id.brick_phiro_stop_motor_spinner,
+				R.string.phiro_motor_both);
+
+		checkIfBrickShowsText(PhiroPlayToneBrick.class, R.string.phiro_play_tone);
+		checkIfBrickShowsText(PhiroPlayToneBrick.class, R.string.phiro_tone_duration);
+		checkIfBrickShowsText(PhiroPlayToneBrick.class, R.string.phiro_seconds);
+		checkIfBrickShowsText(PhiroPlayToneBrick.class, "1");
+		checkIfBrickShowsSpinnerWithEditTextOverlayWithText(PhiroPlayToneBrick.class,
+				R.id.brick_phiro_select_tone_spinner,
+				R.string.phiro_tone_do);
+
+		checkIfBrickShowsText(PhiroRGBLightBrick.class, R.string.brick_phiro_rgb_led_action);
+		checkIfBrickShowsText(PhiroRGBLightBrick.class, R.string.phiro_rgb_led_red);
+		checkIfBrickShowsText(PhiroRGBLightBrick.class, R.string.phiro_rgb_led_green);
+		checkIfBrickShowsText(PhiroRGBLightBrick.class, R.string.phiro_rgb_led_blue);
+		checkIfBrickShowsText(PhiroRGBLightBrick.class, "0");
+		checkIfBrickShowsSpinnerWithEditTextOverlayWithText(PhiroRGBLightBrick.class,
+				R.id.brick_phiro_rgb_light_spinner,
+				R.string.phiro_motor_both);
+
+		checkIfBrickShowsText(PhiroIfLogicBeginBrick.class, R.string.brick_phiro_sensor_begin);
+		checkIfBrickShowsText(PhiroIfLogicBeginBrick.class, R.string.brick_phiro_sensor_second_part);
+		checkIfBrickShowsSpinnerWithEditTextOverlayWithText(PhiroIfLogicBeginBrick.class,
+				R.id.brick_phiro_sensor_action_spinner,
+				R.string.phiro_sensor_front_left);
+
+		checkIfBrickAtPositionShowsText(SetVariableBrick.class, 0, R.string.brick_set_variable);
```

Comment:
also it is unclear which bricks are tested here. so if theres a good reason to use SetVariableBrick.class to reference at least add a comment to state which brick this is about.